extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -140,7 +140,7 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
           }
         } else {
           await this.doUploadDraftWithLocalStorageFallback(mimeMsg, msgData, async () => {
-            await this.view.emailProvider.draftUpdate(this.view.draftId, mimeMsg);
+            await this.view.emailProvider.draftUpdate(this.view.draftId, mimeMsg, this.view.threadId);
```

Comment:
Sometimes you smash your head the whole day just because someone at Google forgot to document [here](https://developers.google.com/gmail/api/reference/rest/v1/users.drafts/update) that `threadId` should be passed to  `.draftUpdate()`. Otherwise, it'll be changed after updating the draft (I still have no idea why).